### ISSUE TITLE
Redis tracing missing after upgrading to redis-rb 5.x

### ISSUE THREAD

**Issue by mscrivo** [DataDog/dd-trace-rb#2373]
**Current behaviour**
Redis tracing is missing after upgrading from redis-rb gem 4.x to 5.x

**Expected behaviour**
Redis should be auto instrumented and traced as it does with 4.x of the gem.

**Steps to reproduce**
All we did was upgrade to redis-rb 5.0.5 and noticed that redis spans stopped coming in.

**Environment**

* **ddtrace version:** 0.54 & 1.6.0 which we just upgraded to
* **Configuration block (`Datadog.configure ...`):**
```ruby
Datadog.configure do |c|
  c.agent.host = <host>
  c.agent.port = 8126

  c.tracing.partial_flush.enabled = ENABLE_PARTIAL_FLUSH

  c.tracing.sampler =
    Datadog::Tracing::Sampling::PrioritySampler.new(
      post_sampler:
        Datadog::Tracing::Sampling::RuleSampler.new(
          [
            # Sample all traces at configured SAMPLE_RATE
            Datadog::Tracing::Sampling::SimpleRule.new(sample_rate: SAMPLE_RATE)
          ]
        )
    )

  c.tracing.analytics.enabled = true
  c.runtime_metrics.enabled = true
  c.runtime_metrics.statsd = Affinity::Monitoring::Statsd.instance
end
```

* **Ruby version:** 3.1.2
* **Operating system:** ubuntu 20.04
* **Relevant library versions:** sidekiq 7.0.1 & redis-rb 5.0.5

**Comment by TonyCTHsu** [comment]
👋 @mscrivo , yup, `redis-rb` 5 is more complicated and we are working on supporting it in the near future.

**Comment by ivoanjo** [comment]
Hey @mscrivo we've added support for redis 5 in #2428 and just released it as dd-trace-rb 1.8.0. Thanks for the report, and let us know if we can help in any other way :)

**Comment by mscrivo** [comment]
Amazing! Thanks so much
